generate.py

```diff
@@ -42,6 +44,18 @@ def write_wav(waveform, sample_rate, filename):
     librosa.output.write_wav(filename, y, sample_rate)
     print('Updated wav file at {}'.format(filename))
 
+def create_seed(filename, sample_rate, quantization_channels, percent=50):
```

Comment:
I agree, I think including a seed larger than `args.window` would be a bit superfluous anyway